pinot-core/src/main/java/org/apache/pinot/core/minion/RawIndexConverter.java

```diff
@@ -204,6 +204,11 @@ private void convertColumn(FieldSpec fieldSpec)
     // Create the raw index
     DataSource dataSource = _originalImmutableSegment.getDataSource(columnName);
     ForwardIndexReader reader = dataSource.getForwardIndex();
+    if (reader == null) {
```

Comment:
This is a refactoring change (since the original variable was named `reader`). Is it alright if I do this as a follow up to keep this PR simpler? I can check for this in a bunch of other files too then.

Reply:
done